### DISCUSSION TITLE
Brotli support

### DISCUSSION THREAD

**Discussion by eweitz** [101arrowz/fflate#127] [Ideas]
fflate is a remarkable library, and Brotli gives [better compression](https://cran.r-project.org/web/packages/brotli/vignettes/brotli-2015-09-22.pdf) than other techniques for common cases.  

How feasible would it be to add Brotli support in fflate?

**Comment by 101arrowz** [comment]
Not super feasible, unfortunately. Brotli is very similar to DEFLATE but it's so slow that it's impractical for real-time use. Moreover, beyond a certain level of complexity, WASM is faster than JS, and I expect Emscripten'd Brotli would be faster than `fflate`.

Brotli decompression is much more feasible because it is as fast as DEFLATE, but then again if you're looking for great compression ratios you may want to look into `fzstd`.

**Reply by eweitz** [reply]
Thanks for the insight!
